content/applications/finance/payment_acquirers/authorize.rst

```diff
@@ -115,6 +115,91 @@ order is canceled, you can click on *Void Transaction* to unlock the funds from
+
+.. image:: authorize/authorize-report1.png
+   :align: center
+   :alt: Select all transactions for the desired range on Authorize.net, and copy them into
+         'report1' of the Excel template.
+
+.. example::
+   .. image:: authorize/authorize-settlement-batch.png
+      :align: center
+      :alt: Settlement batch of the an Authorize.Net statement
+
+   In this case, the first batch (01/01/2021) of the year belongs to the settlement of the
+   31/12/2020, so the **opening** settlement is from the 31/12/2020.
+
+Once the data has been pasted into the :guilabel:`Report 1 Download` sheet, go to
+:menuselection:`Authorize.net --> Transaction Search --> Search for a Transaction`, enter the
+previously used range of batch settlement dates, and click :guilabel:`Search`.
+
+When the list has been generated, click :guilabel:`Download to File`. In the pop-up window, select
+:guilabel:`Expanded Fields with CAVV Response/Comma Separated`, enable :guilabel:`Include Column
+Headings`, and finally click :guilabel:`Submit`. Open the text file, select :guilabel:`All`, copy
+the data and paste it into the :guilabel:`Report 2 Download` sheet of the Excel file.
+
+Transit lines are automatically filled-in and updated in the :guilabel:`transit for report 1` and
+:guilabel:`transit for report 2` sheets of the Excel file. Make sure all entries are present, and if
+not, copy the **formula** from previous filled-in lines of :guilabel:`transit for report 1` or
+:guilabel:`2` and paste it into the empty lines.
+
+.. important::
+   To get the same closing balance, do *not* remove any line from the Excel sheets.
+
+Import into Odoo
+----------------
+
+To import the data into Odoo, go to the Excel file, copy the data from the :guilabel:`transit
+report 2` sheet and **paste special** only the **values** in the :guilabel:`Odoo Import to CSV`
+sheet. Then, look for *blue*-highlighted cells in the :guilabel:`Odoo Import to CSV` sheet. These
+are **chargeback** entries without any **reference** number. To fix this, go to
+:menuselection:`Authorize.Net --> Account --> Statements --> eCheck.Net Settlement Statement`, look
+for :guilabel:`Charge Transaction/Chargeback`, and click it. Copy the **invoice description**, paste
+it into the :guilabel:`Label` cell of the :guilabel:`Odoo Import to CSV` sheet, and add
+"**Chargeback /**" before the description. If you have multiple invoices, add a line into the Excel
+sheet for each invoice and copy/paste the description into each respective :guilabel:`Label` line.
+
+.. note::
+   For **chargeback/returns** combined in the payouts, you need to create a new Excel line for each
+   invoice.
+
+.. example::
+   .. image:: authorize/authorize-chargeback-desc.png
+      :align: center
+      :alt: Charge back description
+
+Next, remove **zero transaction** and **void transaction** line items, and change the **format** of
+the :guilabel:`Amount` column in the :guilabel:`Odoo Import to CSV` sheet to *Number*. Go back to
```

Comment:
I believe it could be considered a UI element. Although it's not a software menu or button, it's not a value (which would be formatted as `value`), and the customer won't modify the heading in any case. Since it's rather static and referred to multiple times to indicate where to input values, I'd have it as guilabel.

I change  the "Number" formatting